### PR TITLE
Remove old BSD links

### LINKED ISSUE #2213: Remove LibreOffice for Android and iOS/iPadOS links
### Affected page

https://www.privacyguides.org/en/productivity/#office-suites

### Description

There are no android Android or iOS/iPadOS version of LibreOffice.
Instead iWork suite can be recommended for iOS/iPadOS.
Related: https://github.com/privacyguides/privacyguides.org/pull/2150#issuecomment-1612387148

### Sources

https://www.libreoffice.org/download/android-and-ios/

### Before submitting

- [X] I am reporting something that is verifiably incorrect, not a suggestion or opinion.
- [X] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### PR DESCRIPTION
**Pull request by mfwmyfacewhen**
Changes proposed in this PR:

- Remove links to freshports

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

File: docs/productivity.md [hunk 1]
```diff
@@ -92,13 +91,10 @@ Our best-case criteria represents what we would like to see from the perfect pro
 
     ??? downloads
 
-        - [:simple-googleplay: Google Play](https://www.libreoffice.org/download/android-and-ios/)
-        - [:simple-appstore: App Store](https://www.libreoffice.org/download/android-and-ios/)
```
Comment (jonaharagon) [inline docs/productivity.md:96]: I disagree with the GitHub Issue that these are incorrectly listed. Collabora is their official ecosystem partner and these are the official ways to edit LibreOffice documents on iOS and Android.

**Review by jonaharagon — DISMISSED** [review]

File: docs/productivity.md (hunk 1 above)
Comment (blacklight447) [reply docs/productivity.md:96]: I agree.

Comment (mfwmyfacewhen) [reply docs/productivity.md:96]: Alright, I just thought since it's a different app it might be a bit confusing but I can re add those.

Comment (dngray) [reply docs/productivity.md]: I think this is resolved now and I'm not seeing any others.

**Review by dngray — APPROVED** [review]

**Review by freddy-m — APPROVED** [review]
LGTM

**Pull request merged**